sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusHostBuilderExtensions.cs

```diff
@@ -44,6 +44,37 @@ public static IWebJobsBuilder AddServiceBus(this IWebJobsBuilder builder, Action
                         config[Constants.DefaultConnectionSettingStringName];
 
                     IConfigurationSection section = config.GetSection(path);
+
+                    bool? autoCompleteMessages = section.GetValue(
```

Comment:
This is a bit of a tricky situation as in the previous version of the SDK, autoComplete could be specified in three separate places. Attempting to just use one common flag here so we somewhat arbitrarily order the precedence from the old config.

Reply:
It applied to separate operations - one for sessions, one for non-sessions, and one for batch (both sessions and non-sessions). So there isn't really a way to respect the old setting if we stick with the current design.